UnitedIncome/components
# [APP-5202] MultiColumn Tile

### Description
> Please provide a brief description of your changes below.

This creates a MultiColumn Tile which represents data:

- in a single column on small devices, and
- in two columns on medium to large devices

This layout is primarily for spending goals, but was converted to a generic component as the layout could be utitlized for other use cases.

NB - The [Spending Goals(APP-5178)](https://unitedincome.atlassian.net/browse/app-5178) use case was created as a story(in the component library) for this MultiColumn Tile component so that it can be visually tested as well as facilitate feedback for the approach taken to achieve the spending goals layout illustrated by the comps in [APP-5202](https://unitedincome.atlassian.net/browse/app-5202) and [Spending Goals(APP-5178)](https://unitedincome.atlassian.net/browse/app-5178). The approach taken would then be ported to the client to complete the creation of the Spending Needs Component(which will utilize this MultiColumn Tile component).

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

1. Verify that the layout displays:

- a single column for small devices
- two columns for medium devices

Also ensure that the above layouts are consistent when pagination occurs

2. Verify the visual consistency of Spending Needs in relation to the comps.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [APP-5202](https://unitedincome.atlassian.net/browse/app-5202)
